src/functions/game/onPlayedTileInteraction.ts

```diff
@@ -105,12 +121,18 @@ export const compareTileInteractionAndSendUpdate = async (
     }
   }
 
-  const finalWsPayload: InteractionSuccessPayload = tripletOrQuadPayload || consecutivePayload;
-  console.log('Triplet Or Quad payload:', tripletOrQuadPayload);
-  console.log('Consecutive payload:', consecutivePayload);
-  console.log('Final INTERACTION_SUCCESS payload:', finalWsPayload);
+  if (canWinGame && JSON.stringify(winGamePayload) !== '{}') {
```

Comment:
Is canWinGame just an additional check here in case ```JSON.stringify(winGamePayload) !== '{}'```  doesn't work? May not need canWinGame if it's not used else where but for readability, it's perfectly fine

Reply:
Yes.
`canWinGame` is a flag to show there is a `MeldEnum.WIN` in the `interactions` array
and
`JSON.stringify(winGamePayload) !== '{}'` is just a hack to check winGamePayload is not an empty object 😂 

So, all together to ensure the backend can send **INTERACTION_SUCCESS** response to client with **WIN** as  the `meldType`